uint16    HandLeft
uint16    HandRight
